backend/pkg/model/model.go

```diff
@@ -30,3 +30,11 @@ type Backend struct {
 	Access     string        `json:"access,omitempty" bson:"access,omitempty"`
 	Security   string        `json:"security,omitempty" bson:"security,omitempty"`
 }
+
+type Tier struct {
+	Id        bson.ObjectId `json:"id,omitempty" bson:"_id,omitempty"`
+	TenantId  string        `json:"tenantId,omitempty" bson:"tenantId,omitempty"`
+	UserId    string        `json:"userId,omitempty" bson:"userId,omitempty"`
+	Name      string        `json:"name,omitempty" bson:"name,omitempty"`
+	BackendId string        `json:"type,omitempty" bson:"type,omitempty"`
```

Comment:
Instead of BackendId, make it Backends and data type would be list.
Ex: Backends []string

Accordingly make changes to all the places

Reply:
changed